src/librustc_trans/adt.rs

```diff
@@ -87,6 +87,14 @@ impl MaybeSizedValue {
     pub fn has_meta(&self) -> bool {
         !self.meta.is_null()
     }
+
+    pub fn to_args(&self) -> Vec<ValueRef> {
```

Comment:
Unnecessary allocation IMO. I'd rather have some helper that takes a `MaybeSizedValue` and drops it.